packages/devtools-modules/client/shared/components/splitter/SplitBox.js

```diff
@@ -100,54 +101,30 @@ const SplitBox = React.createClass({
     splitBox.classList.remove("dragging");
   },
 
-  screenX() {
-    // NOTE: in practice the window might have a border which calls for comparing window.outerWidth and window.innerWidth
-    return window.screenX;
-  },
-
-  screenY() {
-    // NOTE: in practice the window might have a border which calls for comparing window.outerHeight and window.innerHeight
-    return window.screenY;
-  },
-
   /**
    * Adjust size of the controlled panel. Depending on the current
    * orientation we either remember the width or height of
    * the splitter box.
    */
-  onMove(x, y) {
+  onMove({ movementX, movementY }) {
     const node = ReactDOM.findDOMNode(this);
     const doc = node.ownerDocument;
-    const win = doc.defaultView;
 
-    let size;
-    let { endPanelControl } = this.props;
+    if (this.props.endPanelControl) {
+      // For the end panel we need to increase the width/height when the
+      // movement is towards the left/top.
+      movementX = -movementX;
+      movementY = -movementY;
+    }
```

Comment:
likely need the same operation for RTL.

I think we can make RTL work just magically but maybe in a separate patch.